acceptance/bundle/templates/dbt-sql/output.txt

```diff
@@ -22,6 +22,7 @@ Workspace:
 Validation OK!
 
 >>> $CLI bundle validate -t prod
+Warn: target with 'mode: production' should specify an explicit 'targets.prod.git' configuration
```

Comment:
This is not actionable and the warning can be silenced.

Reply:
Why not actionable? You can add git.branch to silence it. 

Context: https://github.com/databricks/cli/commit/a5b86093e#diff-9eedd6c63465cd766c89497eda7f212c5b6868054393cfe464d8d1bc69692576R115 cc @lennartkats-db

Reply:
Okay, sounds like there is no need for this. 

This warning was the only use case we had for bundle.git.inferred, so I've cleaned up that as well.

Please review https://github.com/databricks/cli/pull/2258